planner/src/main/codegen/includes/show.ftl

```diff
@@ -46,21 +46,33 @@ SqlNode SqlShowColumns() :
     }
 }
 
-// DESCRIBE "table"
+// DESCRIBE TABLE "table"
 SqlNode SqlDescribeTable() :
 {
     final Span s;
     final SqlIdentifier schemaName;
     final SqlIdentifier tableName;
 }
 {
-    <DESCRIBE> { s = span(); }
+    <DESCRIBE> { s = span(); } <TABLE>
     tableName = CompoundTableIdentifier()
     {
         return new SqlShowColumns(s.end(this), tableName);
     }
 }
-
+// DESCRIBE MODEL "model_name"
+SqlNode SqlDescribeModel() :
+{
+    final Span s;
+    final SqlIdentifier modelName;
+}
+{
+    <DESCRIBE> { s = span(); } <MODEL>
+    modelName = CompoundTableIdentifier()
```

Comment:
You can replace the `<MODEL> modelName` with `ModelIdentifier`, like e.g. in `SqlPredictModel` in `model.ftl`. With that, you can also use code similar as in https://github.com/nils-braun/dask-sql/blob/main/dask_sql/physical/rel/custom/predict.py#L52 in your python code without the need for a custom function. This is not a must and your code will also work, but I think it might be a bit inconsistent between a call to `PREDICT` and a call to `DESCRIBE MODEL` - as one of them will allow for a compound identifier (which is not used anyway) and the other will not. Also, it will probably make the custom function `get_model_from_compound_identifier` unnecessary (or there is another reason for having a compound identifier which I miss?)

Reply:
And it will automatically fix the missing coverage, as the function will be gone then ;-)

Reply:
GOT it, makes perfect sense, I have replaced type of modelName as modelIdentifier and made changes in python code as well , but currently blocked by above parsing issue .

Reply:
Great!